plugins/modules/aci_l3out_logical_node.py

```diff
@@ -243,6 +248,7 @@ def main():
         node_id=dict(type="int"),
         router_id=dict(type="str"),
         router_id_as_loopback=dict(type="str", default="yes", choices=["yes", "no"]),
+        loopback_address=dict(type="str"),
```

Comment:
Should we use just `loopback`, or set `loopback` as alias.
```suggestion
        loopback_address=dict(type="str", aliases=["loopback"]),
```

Reply:
setting` loopback` as an alias sounds better.